map/src/drawer/components/tracks/LocalClientTrackItem.jsx

```diff
@@ -0,0 +1,69 @@
+import React, {useContext, useState} from "react";
+import AppContext from "../../../context/AppContext";
+import {ListItemText, MenuItem, Switch, Tooltip, Typography} from "@mui/material";
+import LocalClientTrackUtils from "../../util/LocalClientTrackUtils";
+import {BaseBuilder, buildGPX} from "gpx-builder";
+import {Metadata, Point} from "gpx-builder/dist/builder/BaseBuilder/models";
+
+export default function LocalClientTrackItem({track, index}) {
+
+    const ctx = useContext(AppContext);
+    const [indexTrack, setIndexTrack] = useState(index);
+
+    function enableLayer(visible) {
+        if (!visible) {
+            deleteTrackFromMap();
+        } else {
+            addTrackToMap();
+        }
+        LocalClientTrackUtils.saveToLocalStorage(ctx.localClientsTracks);
```

Comment:
? should be context passed to LocalTrackStorage as parameter or accessed as a member ?

Reply:
`+`